pulsar-broker/src/main/java/org/apache/pulsar/PulsarVersionStarter.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar;
+
+/**
+ * Pulsar version entry point.
+ */
+public class PulsarVersionStarter {
```

Comment:
Do we need this change?

Reply:
Yes, I found that other commands such as `pulsar broker`, `pulsar proxy` are a public class with a `main` function, `pulsar version`, as a separate command, also needs to be a class with a `main` function, otherwise, it needs to be a subcommand of other commands